integration

```diff
@@ -787,7 +787,6 @@
   "wimb0/home-assistant-saj-modbus",
   "wizmo2/zidoo-player",
   "wlcrs/huawei_solar",
-  "WolfRevo/climate.spzb0001_thermostat",
```

Comment:
This is missing required changes <https://hacs.xyz/docs/publish/remove#remove-a-repository>

Reply:
Done in https://github.com/hacs/default/commit/8c8a5e792ce03d5c976aa6a5eeb69c2305e6e4af